docs/components.rst

```diff
@@ -0,0 +1,6849 @@
+Components Cross-Reference
```

Comment:
@TripleJQK this file is generated automatically by the doc build process and should not be checked into the repo.  Please remove it from your PR.

Reply:
I also think it is a automatic created file, as I have never modify this file by hands. When did the git status, I was also a little surprised that why the file is in the list.... I will remove this